db/change_cache.go

```diff
@@ -436,16 +436,18 @@ func (c *changeCache) DocChanged(event sgbucket.FeedEvent, docType DocumentType)
 	c.db.DbStats.Database().DCPReceivedCount.Add(1)
 
 	// If the doc update wasted any sequences due to conflicts, add empty entries for them:
-	for _, seq := range syncData.UnusedSequences {
-		base.InfofCtx(ctx, base.KeyCache, "Received unused #%d in unused_sequences property for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
-		change := &LogEntry{
-			Sequence:       seq,
-			TimeReceived:   timeReceived,
-			CollectionID:   event.CollectionID,
-			UnusedSequence: true,
+	if len(syncData.UnusedSequences) > 0 {
```

Comment:
This if statement is not necessary, if `syncData.UnusedSequences` is empty, then the for loop will no-op

Reply:
Need this here so we don't log when the list is empty